Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+sphinx-autobuild = "2020.9.1"
+sphinx-rtd-theme = "0.5.0"
+recommonmark = "0.6.0"
+"sphinxcontrib.httpdomain" = "1.7.0"
+sphinx-tabs = "1.3.0"
+
+[requires]
+python_version = "3"
```

Comment:
Since sphinx requires a minimum of 3.5, just edit this to be "3.5", also update the readme

Reply:
Sadly, pipenv doesn't support minimum python versions. If we make it 3.5 then any python version above that will give us an error :(

Reply:
Yeah you're right my b